.github/workflows/build-beta.yml

```diff
@@ -25,48 +25,28 @@ jobs:
           # Specifying `ref` ensures that the head commit is checked out directly.
           ref: ${{ env.HEAD_COMMIT_HASH }}
 
-      - name: Needs beta build
-        if: ${{ env.BRANCH != 'master' }}
-        id: needs-beta-build
-        run: |
-          version="${BRANCH/Version-v/}"
-          commit_message=$(git show -s --format=%s HEAD)
-          beta_version_regex="Version v[0-9]+\.[0-9]+\.[0-9]+-beta\.[0-9]+"
-
-          if [[ "$commit_message" =~ $beta_version_regex ]]; then
-              printf '%s\n' "Creating a build for $version with $commit_message"
-              echo "NEEDS_BETA_BUILD=true" >> "$GITHUB_OUTPUT"
-          else
-              printf '%s\n' 'Commit message does not match commit message for beta pattern; skipping beta build'
-              echo "NEEDS_BETA_BUILD=false" >> "$GITHUB_OUTPUT"
-          fi
-
       - name: Checkout and setup high risk environment
-        if: ${{ steps.needs-beta-build.outputs.NEEDS_BETA_BUILD == 'true' }}
         uses: MetaMask/action-checkout-and-setup@v1
         with:
           is-high-risk-environment: true
           ref: ${{ env.HEAD_COMMIT_HASH }}
           skip-allow-scripts: true
 
       - name: Run beta build
-        if: ${{ steps.needs-beta-build.outputs.NEEDS_BETA_BUILD == 'true' }}
         env:
-          INFURA_PROJECT_ID: ${{ vars.INFURA_PROJECT_ID }}
-          INFURA_BETA_PROJECT_ID: ${{ vars.INFURA_BETA_PROJECT_ID }}
+          INFURA_PROJECT_ID: ''
```

Comment:
We need a real project ID here, or the beta build won't work. I've just added the test Infura project ID to GHA as the secret `INFURA_PROJECT_ID`, we can update this line to reference that.

Since this is a reusable workflow, I believe we'll need to add it as a parameter as well, and explicitly pass the secret in where this workflow is called.

Reply:
Done